### DISCUSSION TITLE
Expose unique identifier of WebsoketPeer connection

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [godotengine/godot-proposals#8709] [General Discussions]
Currently there seems to be no way of uniquely identifying a [`WebSocketPeer`](https://docs.godotengine.org/en/stable/classes/class_websocketpeer.html), usually WS implementations provide unique ID for any given connection, which makes a lot of logic and handling much easier.

**Comment by Faless** [comment]
If you want a unique ID of the local instance you can use [`get_instance_id`](https://docs.godotengine.org/en/stable/classes/class_object.html#class-object-method-get-instance-id)
